container/spi/src/main/java/org/jboss/arquillian/container/spi/client/protocol/metadata/HTTPContext.java

```diff
@@ -31,21 +31,40 @@
 public class HTTPContext extends NamedContext {
     private final String host;
     private final int port;
+    private final boolean secure;
 
     private final Set<Servlet> servlets;
 
     public HTTPContext(String host, int port) {
-        this("no-named", host, port);
+        this("no-named", host, port, false);
```

Comment:
nitpick - `unnamed` instead of `no-named`?

Reply:
I only went with "no-named" because that's what it was before. We can change it if you'd prefer, but it could potentially break consumers.

Reply:
I thought it was harmless name no one assumed anything on :) But if that's a concern then let's skip it.